contextTree.sh

```diff
@@ -1187,21 +1751,24 @@ function process_cmdb() {
 
 function upgrade_cmdb() {
   local root_dir="$1";shift
+  local gen3_version="$1";shift
```

Comment:
Where do we get this from?

Reply:
TBD - so far its not provided. It is only really needed if we need to constrain the update.